.github/workflows/detect-changes.yml

```diff
@@ -12,33 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# DataHub UI Frontend and BFF
-name: DH Detect changes
-
-# DESCRIPTION:
-# This workflow will determine which categories of jobs should be executed,
-# based on detected changes.
-# It should be used by the 'dh-ci.yml' and 'dh-cd.yml' to ensure they both use
-# the same path's to detect changes.
+name: Detect changes
 
 on:
   workflow_call:
     outputs:
-      dotnet:
-        value: ${{ jobs.changes.outputs.dotnet }}
-      frontend:
-        value: ${{ jobs.changes.outputs.frontend }}
+      dh_dotnet:
+        value: ${{ jobs.changes.outputs.dh_dotnet }}
+      dh_frontend:
+        value: ${{ jobs.changes.outputs.dh_frontend }}
+      watt_frontend:
+        value: ${{ jobs.changes.outputs.watt_frontend }}
 
 jobs:
   changes:
     name: Determine relevant jobs
     runs-on: ubuntu-latest
     # Set job outputs to values from filter step
     outputs:
-      dotnet: ${{ steps.filter.outputs.dotnet }}
-      frontend: ${{ steps.affected.outputs.is-affected }}
+      dh_dotnet: ${{ steps.filter.outputs.dh_dotnet }}
+      dh_frontend: ${{ steps.dh_affected.outputs.is-affected }}
+      watt_frontend: ${{ steps.watt_affected.outputs.is-affected }}
```

Comment:
won't these two not always evaluate with the same result? If that's the case, should it be one output instead?

Reply:
```
- name: Is DH affected?
     id: dh_affected
     uses: ./.github/actions/affected
     with:
       base: ${{ github.ref == 'refs/heads/main' && 'HEAD~1' || format('origin/{0}', env.BASE_BRANCH) }}
       project: app-dh

- name: Is Watt affected?
  id: watt_affected
  uses: ./.github/actions/affected
  with:
    base: ${{ github.ref == 'refs/heads/main' && 'HEAD~1' || format('origin/{0}', env.BASE_BRANCH) }}
    project: ui-watt
```